experimental/dds/tree2/src/test/tree/anchorSet.spec.ts

```diff
@@ -559,6 +658,14 @@ function makePath(...steps: [PathStep, ...PathStep[]]): UpPath {
 	) as UpPath;
 }
 
+function makeFieldPath(
```

Comment:
This and `makeFieldDelta` came about after chatting with @yann-achard-MS on how to construct a move delta.